app/Http/Requests/Api/Auth/RegisterRequest.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace App\Http\Requests\Api\Auth;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class RegisterRequest extends FormRequest
+{
+    public function rules()
+    {
+        return [
+            'name' => 'required|string|max:255',
+            'email' => 'required|string|email|max:255|unique:users',
+            'password' => 'required|string|min:6|confirmed|regex:/^(?=.*?[A-Z])(?=.*?[a-z])(?=.*?[0-9]).{6,}$/',
+            'phone_number' => 'required|string',
+            'dob' => 'required|date',
+            'image' => 'sometimes|image|mimes:png,jpg,jpeg',
```

Comment:
I believe this should be profile_img not image

Reply:
resolved